src/scripts/scene.ts

```diff
@@ -1,10 +1,22 @@
+//import { Spine } from 'pixi-spine';
 import { Assets, Container } from 'pixi.js'
+import { Grid } from './core/models/grid';
 
 export class Scene extends Container {
+    private viewWidth: number;
+    private viewHeight: number;
+    //private spine!: Spine;
     private assets: any[] = [];
+    private gridWidth: number;
 
     constructor(width: number, height: number) {
         super();
+        this.viewWidth = width;
+        this.viewHeight = height;
+
+        this.gridWidth = 0;
+        if(this.viewWidth <= this.height){this.gridWidth = this.viewWidth;}
```

Comment:
For readability can this be laid out over a couple more lines?

For example

`if (this.viewWidth <= this.height) {
   
   ...

} 
else {
   
   ...

}`